internal/handlers/handler_test.go

```
@@ -417,6 +443,10 @@ func TestFind(t *testing.T) {
 						"$size", int32(2),
 					)),
 				)),
+				"sort", types.MustNewDocument(
+					"name", int32(1),
+				),
+				"limit", int32(1),
```

Why? I think that could hide errors